frontend/src/workspaces/features/InviteMemberDialog/InviteMemberDialog.tsx

```diff
@@ -26,7 +27,7 @@ type InviteMemberDialogProps = {
 };
 
 type Form = {
-  user: User | null;
+  user: UserPicker_UserFragment | null;
```

Comment:
Instead of relying on the presence of this fragment, I tend to use the type of the prop directly
```
user: React.ComponentProps<typeof UserPicker>["user"]
```